style.css

```diff
@@ -0,0 +1,268 @@
+    width: 280px;
+    color: white;
+}
+
+.about {
+    margin-top: 30px;
+    margin-right: 30px;
+    margin-bottom: 30px;
+    margin-left: 15px;
+    font-size: 20px;
+    font-weight: bolder;
+}
+
+.about-text {
+    margin-bottom: 20px;
+    margin-left: 15px;
+    margin-right: 15px;
+}
+
+.about-additional {
+    margin-left: 15px;
+    margin-right: 15px;
+}
+
+.education {
+    margin-top: 70px;
+    font-size: 20px;
+    font-weight: bolder;
+}
+
+.university {
+    font-family: "Encode Sans", sans-serif;
+    margin-left: 15px;
+}
+
+.experience {
+    margin-bottom: 20px;
+    margin-left: 15px;
+    margin-top: 30px;
+    font-size: 20px;
+    font-weight: bolder;
+}
+
+.experience-text {
+    margin-bottom: 20px;
+    margin-left: 15px;
+}
+
+.skills {
+    margin-top: 70px;
+    margin-bottom: 20px;
+    font-size: 20px;
+    font-weight: bolder;
+    margin-left: 15px;
+}
+
+.skills-text {
+    margin-bottom: 40px;
+    margin-left: 15px;
+    margin-right: 30px;
+}
+
+.photo img {
+    height: 100%;
+    width: 100%;
+    display: flex;
+}
+
+.email {
+    font-family: "Encode Sans", sans-serif;
+    display: flex;
+    font-size: 22px;
+}
+
+.progress-bar-front80 {
+    height:5px;
+    width:80%;
+    background-color: white;
+}
+
+.progress-bar-front75 {
+    height: 5px;
+    width: 75%;
+    background-color: white;
+}
+
+.progress-bar-front90 {
+    height: 5px;
+    width: 90%;
+    background-color: white;
+}
+
+.progress-bar-back {
+    height:5px;
+    width:100px;
+    background-color: grey;
+}
+
+.skills-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 5px;
+}
+
+.contact-section {
+    align-items: center;
+    flex-direction: column;
+    display: flex;
+}
+
+.contact-container {
+    display: flex;
+    width: 500px;
+    height: 180px;
+    justify-content: space-evenly;
+    align-items: center;
+    flex-direction: column;
+    margin-top: 30px;
+}
+
+.high-school {
+    margin-top: 30px;
+}
+
+.education-text {
+    margin-top: 7px;
+    font-size: 16px;
+}
+
+.job-position {
+    margin-bottom: 7px;
+    margin-left: 15px;
+}
+
+.experience-text {
+    font-size: 16px;
+    margin-bottom: 30px;
+    margin-left: 15px;
+    margin-right: 15px;
+}
+
+.education {
+    margin-top: 50px;
+    margin-left: 15px;
+    margin-bottom: 20px;
+}
+
+@media (max-width: 1200px) {
+    .experience {
+        margin-top: 2px;
+    }
+}
+
+@media (max-width: 992px) {
+    .contact-container {
+        width: 400px;
+    }
+    .contact {
+        font-size: 18px;
+    }
+    .phone {
+        font-size: 18px;
+    }
+    .email {
+        font-size: 18px;
+    }
+}
+
+@media (max-width: 768px) {
+    .content {
+        flex-direction: column-reverse;
+    }
+    .left-panel {
+        width: 100%;
+    }
+    .university {
+        margin-bottom: 20px;
+    }
+    .contact {
+        font-size: 18px;
+    }
+    .phone {
+        font-size: 18px;
+    }

```

Comment:
You already have `font-size: 18px;` with `@media (max-width: 992px)` media query. Remember that both media queries are applicable in this case. 
![image](https://github.com/user-attachments/assets/c0122dfd-cf46-4f98-ae7a-360c069573d8)

Reply:
You are totally right! Thank you for highlighting it :)